README.adoc

```diff
@@ -1,3 +1,41 @@
+= C++ example project scanned on SonarCloud using GitLab CI
+// URIs:
+:uri-qg-status: https://sonarcloud.io/dashboard?id=sc-sq-c-family-examples_linux-autotools-gitlab-ci-sc
+:img-qg-status: https://sonarcloud.io/api/project_badges/measure?project=sc-sq-c-family-examples_linux-autotools-gitlab-ci-sc&metric=alert_status
+:uri-build-status: https://gitlab.com/tomasz.kaminski/linux-autotools-gitlab-ci-sc/-/commits/main
+:img-build-status: https://gitlab.com/tomasz.kaminski/linux-autotools-gitlab-ci-sc/badges/main/pipeline.svg
+
+image:{img-build-status}[Build Status, link={uri-build-status}]
+image:{img-qg-status}[Quality Gate Status,link={uri-qg-status}]
+
+*This project is analysed on https://sonarcloud.io/dashboard?id=sc-sq-c-family-examples_linux-autotools-gitlab-ci-sc[SonarCloud]!*
+
+It is very easy to analyze a C/C++/Objective-C project with SonarCloud https://sonarcloud.io/documentation/getting-started/gitlab/[on GitLab CI]:
+
+. Create a `sonar-project.properties` file to store your configuration
+. In your `.gitlab-ci.yml` file:
+.. As the `.pre` stage  stage:
+... Download the https://docs.sonarqube.org/latest/analysis/scan/sonarscanner/[Sonar Scanner]
+... Download the https://docs.sonarqube.org/latest/analysis/languages/cfamily/#header-4[Build Wrapper]
```

Comment:
Turns out you can replace all the SQ documentation with (potentially better) SC docs. See https://github.com/sc-sq-c-family-examples/macos-cmake-gh-actions-sc for example

Reply:
Thanks updated.

Reply:
I don't see the update. Did you update it on gitlab side?
See the links in these steps:
```
Download the Sonar Scanner
Download the Build Wrapper
```
in https://github.com/sc-sq-c-family-examples/windows-msbuild-gh-actions-sc/blob/main/README.adoc

Reply:
Ah, I missed them.